src/main/java/com/inventorysystem/inventory/dto/Vendor.java

```diff
@@ -4,7 +4,7 @@
 
 public @Data
 class Vendor {
-    private int VendorID;
-    private String VendorName;
-    private String PhoneNumber;
+    private int vendorID;
+    private String vendorName;
+    private String phoneNumber;
```

Comment:
Class variables/fields must be lowercase. [Reference](https://www.codejava.net/coding/10-java-core-best-practices-every-java-programmer-should-know#NamingConvention)